### PR TITLE
calife: update to 3.0.6, add modeline

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update to 3.0.6. Add modeline, fix spaces and typography.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.6.8
Xcode 3.2.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by barracuda156** [comment]
Any idea what goes wrong on the bot?
```
--->  calife-3.0.6.tar.xz does not exist in /opt/local/var/macports/distfiles/calife
[108](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:111)
  --->  Attempting to fetch calife-3.0.6.tar.xz from https://distfiles.macports.org/calife
[109](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:112)
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[110](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:113)
                                   Dload  Upload   Total   Spent    Left  Speed
[111](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:114)
  
[112](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:115)
    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[113](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:116)
    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[114](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:117)
  DEBUG: Fetching distfile failed: The requested URL returned error: 404 
[115](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:118)
  --->  Attempting to fetch calife-3.0.6.tar.xz from https://assets.keltia.net/calife
[116](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:119)
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[117](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:120)
                                   Dload  Upload   Total   Spent    Left  Speed
[118](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:121)
  
[119](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:122)
    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0DEBUG: Fetching distfile failed: Could not resolve host: assets.keltia.net
[120](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:123)
  Error: Failed to fetch calife: Could not resolve host: assets.keltia.net
[121](https://github.com/macports/macports-ports/runs/7490475520?check_suite_focus=true#step:8:124)
  DEBUG: Error code: NONE
```

 It certainly works on my end:
```
10:~ svacchanda$ sudo port clean calife
--->  Cleaning calife
10:~ svacchanda$ sudo port -v fetch calife
--->  Fetching distfiles for calife
--->  calife-3.0.6.tar.xz does not exist in /opt/local/var/macports/distfiles/calife
--->  Attempting to fetch calife-3.0.6.tar.xz from http://distfiles.macports.org/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   126    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from http://nue.de.distfiles.macports.org/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   291    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from http://ema.uk.distfiles.macports.org/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   315    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from http://fra.de.distfiles.macports.org/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   279    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from http://mse.uk.distfiles.macports.org/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   196    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from http://cph.dk.distfiles.macports.org/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   196    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from http://jog.id.distfiles.macports.org/macports/distfiles/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   168    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from http://fco.it.distfiles.macports.org/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   146    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from http://cjj.kr.distfiles.macports.org/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   178    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from http://kmq.jp.distfiles.macports.org/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   305    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
--->  Attempting to fetch calife-3.0.6.tar.xz from https://assets.keltia.net/calife
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 98768  100 98768    0     0  47338      0  0:00:02  0:00:02 --:--:-- 47370
10:~ svacchanda$ 
```

**Comment by reneeotten** [comment]
@barracuda156 please do mix functional changes and whitespace changes in one commit.

File: security/calife/Portfile [hunk 1]
```diff
@@ -1,51 +1,53 @@
...
+                    (shell aliases, PATH and so on).
+homepage            https://www.keltia.net/programs/calife
+master_sites        https://assets.keltia.net/calife
+
+checksums           rmd160  92fe1a21d3df98120ce66b696503cffe90472d52 \
+                    sha256  f09b5c73439bab14a7ff1438b8f22a8da6216a5caf0ef8467f85467b2217357e \
+                    size    98768
+use_xz              yes
+
+use_configure       yes
```
Comment (reneeotten) [inline security/calife/Portfile:22]: this is the default so can be removed

File: security/calife/Portfile [hunk 2]
```diff
@@ -1,51 +1,53 @@
...
-  xinstall -m 600 ${worksrcpath}/calife.auth-dist \
-			${destroot}${prefix}/etc
-  xinstall -m 644 ${worksrcpath}/calife.1 \
-			${destroot}${prefix}/share/man/man1
-  xinstall -m 644 ${worksrcpath}/calife.auth.5 \
-			${destroot}${prefix}/share/man/man5
+    xinstall -m 755 -d ${destroot}${prefix}/bin
+    xinstall -m 755 -d ${destroot}${prefix}/share/calife
+    xinstall -m 755 -d ${destroot}${prefix}/share/man/man1
+    xinstall -m 755 -d ${destroot}${prefix}/share/man/man5
```
Comment (reneeotten) [inline security/calife/Portfile:30]: are the "bin" and "man?" directories not made by default already?

**Review by reneeotten — CHANGES_REQUESTED** [review]

**Comment by barracuda156** [comment]
> @barracuda156 please do mix functional changes and whitespace changes in one commit. 

I didn’t forget, but here the only functional change been that of the version number. Which I mentioned explicitly above.
There are no changes to the code.

File: security/calife/Portfile (hunk 2 above)
Comment (barracuda156) [reply security/calife/Portfile:30]: This has been there in the Portfile. I can try removing it, if you suggest.

File: security/calife/Portfile (hunk 1 above)
Comment (barracuda156) [reply security/calife/Portfile:22]: Just the configure line, right?

**Comment by reneeotten** [comment]
> > @barracuda156 please do mix functional changes and whitespace changes in one commit. 
> 
> I didn’t forget, but here the only functional change been that of the version number. Which I mentioned explicitly above.
> There are no changes to the code.

A version change is a functional change.

**Comment by barracuda156** [comment]
> A version change is a functional change.

Ok I will change it.

**Comment by barracuda156** [comment]
@reneeotten Please take a look.

**Review by reneeotten — APPROVED** [review]

**Pull request merged**